cmd/build.go

```diff
@@ -246,6 +266,7 @@ func runBuild(args []string, mode mode, modeArgs []string) {
 		}
 
 		fmt.Println("\nAvailable flags:")
+		fmt.Printf("  %s='%s' [string] // Output directory\n", outputDirFlagName, outputDirName)
 		for name, flag := range genOutput.Flags {
```

Comment:
Please consider changing the output to be sorted alphabetically as it greatly aids readability and usability. Add the following function:
```
func sortMapKeys(m interface{}) []string {
	keys := reflect.ValueOf(m).MapKeys()
	keyList := []string{}

	for _, key := range keys {
		keyList = append(keyList, key.Interface().(string))
	}
	sort.Strings(keyList)
	return keyList
}
```
and then change the code to
```
for _, name := range sortMapKeys(genOutput.Flags) {
			flag := genOutput.Flags[name]
			fmt.Printf("  %s='%s' [%s]", name, flag.Value, flag.Type)
			if len(flag.AllowedValues) > 0 {
				fmt.Printf(" ('%s')", strings.Join(flag.AllowedValues, "', '"))
			}
			if flag.Description != "" {
				fmt.Printf(" // %s", flag.Description)
			}
			fmt.Println()
		}
```

Reply:
done